posts/2023-02-15-hackaton-made-by-cesi.adoc

```diff
@@ -0,0 +1,40 @@
+= Hackaton Made by Cesi
+sachgar
+v1.0, 2023-02-15
+:title: hackaton-made-by-cesi
+:lang: en
+:tags: [hackaton, event, en]
+
+
+On January 28, 2023, the CESI engineering school in La Rochelle hosted a hackathon with an apocalyptic theme. 25 participants gathered in teams of five to imagine innovative solutions to deal with an emergency situation. As with any events, it was essential for us to find sponsors to ensure that the event will be successful. We naturally thought Lunatech would be a good fit for us. They kindly agreed to finance the staff's hoodies and to send us some goodies for all of the participants.
+
+image::../media/2023-02-15-hackaton-made-by-cesi/staff.jpeg[Staff,640,480]
+
+The hackathon started with a presentation of the apocalyptic scenario, followed by a brainstorming session to generate ideas. In order to help participants in their search for ideas, different techniques were used such as the World Cafe. 
```

Comment:
The apocalyptic scenario in details:
```
"Attention all participants of our hackathon! It is the 10th of January 2049! We have received alarming information that an apocalyptic catastrophe is about to happen in our city in the year 2050, according to the predictions of the doomsday clock and several scientists. Scientists estimate that it will happen in the next few months.

Fortunately, we have prepared a secure bunker to protect the participants of the event. However, it is our duty to prepare our city for this catastrophe. This is where you come in!

We need your help to find ways to protect the people and property of our city. You have 24 hours on 28 January to come up with an innovative and workable solution to this disaster.

We are counting on your ingenuity and determination to save our city. Good luck to you all!"
```

Reply:
The wikipedia article is a great addition.